src/frontend/js/widgets/Dashboard/components/SearchResultsCount/index.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { useSearchParams } from 'react-router-dom';
+
+const messages = defineMessages({
+  searchCountText: {
+    defaultMessage:
+      '{nbResults} {nbResults, plural, one {result} other {results}} found for your research',
+    description: 'Text to indicate the total number of results for a research',
+    id: 'Dashboard.components.SearchResultsCount.searchCountText',
+  },
+});
+
+interface SearchResultsCountProps {
+  nbResults?: number;
+}
+
+const SearchResultsCount = ({ nbResults }: SearchResultsCountProps) => {
+  const [searchParams] = useSearchParams();
+  return (
+    <div
+      className={classNames('list__count-description', {
+        'list__count-description--no-results': !searchParams.get('query') || !nbResults,
```

Comment:
What is this class used for?

Reply:
When no research is active, we don't display the counter.